airbyte-workers/src/main/java/io/airbyte/workers/process/AsyncOrchestratorPodProcess.java

```diff
@@ -126,11 +129,14 @@ private int computeExitValue() {
       // we read the status from the Kubernetes API, we need to check the doc store again.
       final AsyncKubePodStatus secondDocStoreStatus = getDocStoreStatus();
       if (secondDocStoreStatus.equals(AsyncKubePodStatus.FAILED)) {
+        log.info("State Store reports orchestrator pod {} failed", getInfo().name());
```

Comment:
@davinchia Same info -> error change?

Reply:
Thank you for noticing these @jdpgrailsdev ! 

I switched all of them to WARN to start with.